### DISCUSSION TITLE
Windows application, sound card

### DISCUSSION THREAD

**Discussion by LarryAndrew** [jamulussoftware/jamulus#3100] [General]
I'm currently getting a snap, crackel, pop noise during a session with one other person on a Jamulus server .  Will an old sound card residing in a desktop, being used by one of us cause noises. Would it help to install a new sound card.

**Comment by ann0see** [comment]
Yes, Soundcards can cause these issues. Also bad connectivity  might cause audio artefacts.

**Comment by dcorson-ticino-com** [comment]
> I'm currently getting a snap, crackel, pop noise 

I would expect that a less than top sound card would add constant hissing, or strange noises when playing high tones.
Snap, crackel, and pop are more likely to be the result of a poor internet connection.
Check first that all the people connected to the server are using a wired connection to the internet, wifi adds delay and very often noise.
Even just one person connected by wifi can pollute the sound of all the others when mixed on the server.

**Comment by pljones** [comment]
> I'm currently getting a snap, crackel, pop noise during a session with one other person on a Jamulus server . Will an old sound card residing in a desktop, being used by one of us cause noises. Would it help to install a new sound card.

As has been said, lots of things contribute to a noisy session.

Does it happen on _every_ server with around the same (short) ping?

Even when you're alone?

Do you have a stable overall delay to servers you use?

Is your CPU struggling at all?

**Comment by LarryAndrew** [comment]
I try to use the lowest ping number I can find. I in the Tampa area. a server in Miami , As the light that shows the delay changes from yellow to green there is always noise, as I play by myself with a backing track.
________________________________

**Reply by pljones** [reply]
Hm, I tried to look up "internet routing Tampa to Miami" on Google and got very few hits.  200 miles should give you a pretty good ping and stable delay - it's about half the distance of London to Frankfurt, which for me is rock solid.

Are you streaming the backing track over the network?  That's not going to help if you've a poor network connection.

Check your WiFi is turned off _everywhere_, preferably including at your router.

I'd still check your CPU stability and whether you computer is tuned properly for real time audio.

**Reply by dcorson-ticino-com** [reply]
As plj suggested, try other servers.
A very low ping is good, but not everything.
I am in Switzerland and sometimes have noise on certain servers in Europe, but there are servers in other parts of the world with pings of over 100ms that are clean as a whistle. (just almost impossible to use because of the delay :) ) 
Looking around I can, however, always find a server that is good, no noise and acceptable delay.
Note that what is important is the regularity with which you receive the Internet packets.  Even if you have very low ping times and a very fast service, if the packets arrive in bunches or out of order you will have noise.  The Jamulus protocol is able to correct out of order packets, but only within limits.
Unfortunately I am not aware of any way that one can influence the switching behaviour of the packet transport across the Internet to get more regular delivery.  The best we can do is try other servers and as a last chance, change Internet providers.
